integration/test/bootstrap/setup.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 
 # Start the bootstrap process.
-make
-./apptestctl bootstrap --kubeconfig="$(kind get kubeconfig)"
+go install .
```

Comment:
Makefile fails because there is no architect. Easier to just go install.

```
make: architect: Command not found
====> apptestctl-v-linux-amd64
CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -ldflags "-w -linkmode 'auto' -extldflags '-static' -X 'github.com/giantswarm/apptestctl/pkg/project.buildTimestamp=2020-11-03T16:40:11Z' -X 'github.com/giantswarm/apptestctl/pkg/project.gitSHA=711902c316eb44faa4281ce8bd6d3306bfdc009d'" -o apptestctl-v-linux-amd64 .
====> apptestctl
cp -a apptestctl-v-linux-amd64 apptestctl
====> build
./integration/test/bootstrap/setup.sh: line 5: apptestctl: command not found
```